tools/dependency/pip_check.py

```diff
@@ -51,44 +43,57 @@ def config_requirements(self) -> set:
     @cached_property
     def dependabot_config(self) -> dict:
         """Parsed dependabot config"""
-        with open(os.path.join(self.path, DEPENDABOT_CONFIG)) as f:
+        with open(os.path.join(self.path, self.dependabot_config_path)) as f:
             return yaml.safe_load(f.read())
 
+    @property
+    def dependabot_config_path(self) -> str:
+        return self._dependabot_config
+
     @cached_property
     def requirements_dirs(self) -> set:
         """Set of found directories in the repo containing requirements.txt"""
         return set(
             root[len(self.path):]
             for root, dirs, files in os.walk(self.path)
-            if "requirements.txt" in files)
+            if self.requirements_filename in files)
+
+    @property
+    def requirements_filename(self) -> str:
+        return self._requirements_filename
 
-    def check_dependabot(self) -> int:
+    def check_dependabot(self) -> None:
         """Check that dependabot config matches requirements.txt files found in repo"""
         missing_dirs = self.config_requirements - self.requirements_dirs
         missing_config = self.requirements_dirs - self.config_requirements
-
+        correct = self.requirements_dirs & self.config_requirements
```

Comment:
`&` or `and`?

Reply:
these are set operations - this a set intersection, and is correct 

i could use the set methods instead i guess

Reply:
I think I would probably find it a bit clearer TBH.